<launch>
    <param name="robot_description"
        command="$(find xacro)/xacro.py $(find turtlebot3_description)/urdf/turtlebot3_burger.urdf.xacro" />


    <!-- <param name="robot_description"
        textfile="$(find testbot_description)/urdf/testbot.urdf" /> -->
    <group ns="robot1">
        <param name="tf_prefix" value="robot1_tf" />
        <include file="$(find multi_robot_tf)/launch/one_robot.launch" >
            <arg name="init_pose" value="-x 1 -y 1 -z 0" />
            <arg name="robot_name" value="Robot1" />
        </include>
    </group>


    <group ns="robot2">
        <param name="tf_prefix" value="robot2_tf" />
        <include file="$(find multi_robot_tf)/launch/one_robot.launch" >
            <arg name="init_pose" value="-x -1 -y 1 -z 0" />
            <arg name="robot_name" value="Robot2" />
        </include>
    </group>
</launch>

    

